content/applications/sales/crm/pipeline/multi_sales_team.rst

```diff
@@ -6,41 +6,41 @@ In Odoo, you can manage several sales teams, departments or channels
 with specific sales processes. To do so, we use the concept of *Sales
 Channel*.
 
-Create a new sales channel
+Create a new sales team
 ==========================
 
-To create a new *Sales Channel*, go to :menuselection:`Configuration --> Sales Channels`.
+To create a new *Sales Team*, go to :menuselection:`Configuration --> Sales Teams`.
 
 There you can set an email alias to it. Every message sent to that email
 address will create a lead/opportunity.
 
 .. image:: media/multi_sales_team01.png
     :align: center
 
-Add members to your sales channel
+Add members to your sales team
 ---------------------------------
 
-You can add members to any channel; that way those members will see the
-pipeline structure of the sales channel when opening it. Any
-lead/opportunity assigned to them will link to the sales channel.
-Therefore, you can only be a member of one channel.
+You can add members to any team; that way those members will see the
+pipeline structure of the sales team when opening it. Any
+lead/opportunity assigned to them will link to the sales team.
+Therefore, you can only be a member of one team.
 
 This will ease the process review of the team manager.
```

Comment:
Let's delete this line as well since it goes with the previous line that we're deleting:

`This will ease the process review of the team manager.`